CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+#### Changed
+
+- Bump cairo to 2.4.0.
```

Comment:
Idk if we should write it in changelog since we will be compatible with a lot of versions of Scarb (and therefore Cairo) due to test collector migration. We could add sth like "migrated test compilation and collection to Scarb, snforge should now be compatible with every Scarb version >= 2.4.0 unless breaking changes happen"

Reply:
Done